### ISSUE TITLE
No loss function 'cosine_proximity'

### ISSUE THREAD

**Issue by mdvertola** [guxd/deep-code-search#54]
There is no Keras loss function called 'cosine_proximity'. This was also not defined as a custom loss function in the codebase, changing to 'cosine_similarity' in main.py fixed for me and allowed the model to begin training.

**Comment by guxd** [comment]
Thank you, but there is no `cosine_proximity` in `main.py`. Do you mean `model.py`?
Feel free to submit a pull request.
